### ISSUE TITLE
Closed tabs aren't actually closed

### ISSUE THREAD

**Issue by Haltroy** [Haltroy/Yorot.Avalonia#23]
It seems that Avalonia can't dispose. Further research says that "there's no need because there's nothing to dispose" but creating a new tab results ~14MB memory whilst closing it doesn't actually free up any memory. My current temporary solutions that I applied are:
 - Make TabWindows IDİsposable & set stuff that can be set to null to null.
 - Navigate Tab to about:blank to free up memory from CEF and also make tab shut its mouth.
 - Remove LoadPage events on SystemSession of that tab and remove it from Systems list, add it to ClosedSystems list and detach SessionSystem from tab.
 - Clean the containers that host the tab content
 - Use WebView's DisposeWith() with the tab itself.
 - Set Content of the tab to null
 - Tell garbage collector to collect at the end.

By the way, none of these actually removed the tab from memory but now when you close a YouTube tab with a video playing now it just won't play it anymore. 

This is gonna cause a big headache, might ask Avalonia experts or the dev team itself on this issue.

**Comment by Haltroy** [comment]
## The good news

Turns out this is a bug with the tab control library anf the creator of that library knows the issue so there's nothing I need to do.

## The bad news

The fix is probably gonna be in the library's version where it requires Avalonia 0.11 which is in preview and planned for release in Q1 2023 (January, February, March) meaning that this bug will be in the indev1 version. Hopefully, if Avalonia 0.11 releases in January this won't be a problem (and by looking at the project, it seems that it's almost done already). If it did, I either have to use the preview version of Avalonia and FluentAvaloniUI or release the indev1 with this bug.

**Comment by Haltroy** [comment]
## The good news

That bug was fixed, my exams hit me again which slowed down the development of Yorot and we are now in Q1 of 2023 so Avalonia 11 might release soon.

Looking at the AvaloniaUI's GitrHub projects page, there's less stuff to do and since Avalonia team is bigger, I think it will be out soon. 

TL;DR, the bug is fixed, and Yorot indev1 release will come with this fixed.
